<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Animation Settings Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Resolution (pixels)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QSpinBox" name="spinBoxWidth" >
       <property name="toolTip" >
        <string>View Width (in pixels) </string>
       </property>
       <property name="statusTip" >
        <string>Sets the width (in pixles) to use while generating animation.</string>
       </property>
       <property name="whatsThis" >
        <string>Sets the width (in pixles) to use while generating animation.</string>
       </property>
       <property name="maximum" >
        <number>8000</number>
       </property>
       <property name="minimum" >
        <number>2</number>
       </property>
       <property name="singleStep" >
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" >
      <widget class="QDoubleSpinBox" name="frameRate" >
       <property name="maximum" >
        <double>999999.000000000000000</double>
       </property>
       <property name="minimum" >
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2" >
      <widget class="QSpinBox" name="spinBoxNumberOfFrames" >
       <property name="maximum" >
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2" >
      <widget class="QDoubleSpinBox" name="animationDuration" >
       <property name="maximum" >
        <double>999999999999999967336168804116691273849533185806555472917961779471295845921727862608739868455469056.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="labelFrameRate" >
       <property name="text" >
        <string>Frame Rate (fps)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2" >
      <widget class="QSpinBox" name="spinBoxFramesPerTimestep" >
       <property name="maximum" >
        <number>2147483647</number>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="labelFramesPerTimestep" >
       <property name="text" >
        <string>No. of Frames / timestep</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" >
      <widget class="QSpinBox" name="spinBoxHeight" >
       <property name="toolTip" >
        <string>View Height (in pixels)</string>
       </property>
       <property name="statusTip" >
        <string>Sets the height (in pixles) to use while generating animation.</string>
       </property>
       <property name="whatsThis" >
        <string>Sets the height (in pixles) to use while generating animation.</string>
       </property>
       <property name="maximum" >
        <number>8000</number>
       </property>
       <property name="minimum" >
        <number>2</number>
       </property>
       <property name="singleStep" >
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="labelAnimationDuration" >
       <property name="text" >
        <string>Animation Duration (sec)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Number Of Frames</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxDisconnect" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="toolTip" >
      <string>&lt;html>Disconnect from the server before saving animation. The server will save the animation and then exit.&lt;/html></string>
     </property>
     <property name="statusTip" >
      <string>&lt;html>Disconnect from the server before saving animation. The server will save the animation and then exit.&lt;/html></string>
     </property>
     <property name="whatsThis" >
      <string>&lt;html>Disconnect from the server before saving animation. The server will save the animation and then exit.&lt;/html></string>
     </property>
     <property name="text" >
      <string>Disconnect before saving animation</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="toolTip" >
        <string>Generate and Save Animation</string>
       </property>
       <property name="text" >
        <string>Save Animation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>animationDuration</tabstop>
  <tabstop>frameRate</tabstop>
  <tabstop>spinBoxFramesPerTimestep</tabstop>
  <tabstop>spinBoxNumberOfFrames</tabstop>
  <tabstop>spinBoxWidth</tabstop>
  <tabstop>spinBoxHeight</tabstop>
  <tabstop>checkBoxDisconnect</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
